src/docs/BUILD.md

```diff
@@ -3,85 +3,200 @@
-    cd openvino.genai
+    export OpenVINO_DIR=<INSTALL_DIR>/runtime
+    export PYTHONPATH=<INSTALL_DIR>/python:./build/:$PYTHONPATH
+    export DYLD_LIBRARY_PATH=<INSTALL_DIR>/runtime/lib/intel64:$LD_LIBRARY_PATH
+    ```
+
+    Windows Command Prompt:
+    ```cmd
+    set OpenVINO_DIR=<INSTALL_DIR>\runtime
+    set PYTHONPATH=<INSTALL_DIR>\python;%CD%\build;%PYTHONPATH%
+    set PATH=<INSTALL_DIR>\runtime\lib\intel64;%PATH%
     ```
-3. Build the project:
+    
+    Windows PowerShell:
+    ```sh
+    $env:OpenVINO_DIR = "<INSTALL_DIR>\runtime"
+    $env:PYTHONPATH = "<INSTALL_DIR>\python;$PWD\build;$env:PYTHONPATH"
+    $env:PATH = "<INSTALL_DIR>\runtime\lib\intel64;$env:PATH"
+    ```
+
+4. Build the project:
     ```sh
-    call <INSTALL_DIR>\setupvars.bat
     cmake -DCMAKE_BUILD_TYPE=Release -S ./ -B ./build/
-    cmake --build ./build/ --config Release --target package -j
-    cmake --install ./build/ --config Release --prefix ov
+    cmake --build ./build/ --config Release -j
     ```
 
-## Build for macOS Systems
+5. Install OpenVINO GenAI:
 
-### Software Requirements
+    #### Option 1 - using cmake:
+    
+    The following command will store built OpenVINO GenAI artifacts along with OpenVINO in `<INSTALL_DIR>`:
 
-- [CMake](https://cmake.org/download/) 3.23 or higher
-- [brew](https://brew.sh/) package manager to install additional dependencies:
     ```sh
-    brew install coreutils scons
+    cmake --install ./build/ --config Release --prefix <INSTALL_DIR>
     ```
-- Clang compiler and other command line tools from Xcode 10.1 or higher:
+
+    #### Option 2 - setting paths to built OpenVINO GenAI artifacts manually:
+
+    The path to the OpenVINO GenAI root directory is referred as `<GENAI_ROOT_DIR>` throughout the document.
+
+    Linux:
     ```sh
-    xcode-select --install
+    export PYTHONPATH=<GENAI_ROOT_DIR>/build/:$PYTHONPATH
+    export LD_LIBRARY_PATH=<GENAI_ROOT_DIR>/build/openvino_genai/:$LD_LIBRARY_PATH
     ```
-- Python 3.8 or higher
 
-### Build Instructions
+    macOS:
+    ```sh
+    export PYTHONPATH=<GENAI_ROOT_DIR>/build:$PYTHONPATH
+    export DYLD_LIBRARY_PATH=<GENAI_ROOT_DIR>/build/openvino_genai:$DYLD_LIBRARY_PATH
+    ```
 
-1. Build and install OpenVINO from sources following the [instructions](https://github.com/openvinotoolkit/openvino/wiki#how-to-build)  
-The path to the openvino install directory is referred as <INSTALL_DIR> throughout the document.
-2. Clone OpenVINO GenAI repository and init submodules:
+    Windows Command Prompt:
+    ```cmd
+    set PYTHONPATH=<GENAI_ROOT_DIR>\build;%PYTHONPATH%
+    set PATH=<GENAI_ROOT_DIR>\build\openvino_genai;%PATH%
+    ```
+
+    Windows PowerShell:
+    ```sh
+    $env:PYTHONPATH = "<GENAI_ROOT_DIR>\build;$env:PYTHONPATH"
+    $env:PATH = "<GENAI_ROOT_DIR>\build\openvino_genai;$env:PATH"
+    ```
+
+To optimize the package size, you can reduce the ICU (International Components for Unicode) data size when building OpenVINO Tokenizers.
```

Comment:
```suggestion
To optimize the package size, you can reduce the ICU (International Components for Unicode) data size when OpenVINO Tokenizers are built as a submodule of OpenVINO GenAI.
```
We don't have Tokenizers described. That makes this statement to look out of blue.